zcash_client_sqlite/src/wallet/scanning.rs

```diff
@@ -453,9 +453,9 @@ pub(crate) fn replace_queue_entries(
             "SELECT block_range_start, block_range_end, priority
             FROM scan_queue
             WHERE (
-                -- the start is contained within the range
+                -- the start is contained within or adjacent to the range
                 :start >= block_range_start
-                AND :start < block_range_end
+                AND :start <= block_range_end
             )
             OR (
                 -- the end is contained within the range
```

Comment:
Is it intended that this merges with a range adjacent to the start but not adjacent to the end?

Reply:
Ah... good point. I'll fix that too.